dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/util/NacosNamingServiceUtils.java

```diff
@@ -91,7 +93,9 @@ public static ServiceInstance toServiceInstance(Instance instance) {
      * @since 2.7.5
      */
     public static String getGroup(URL connectionURL) {
-        return connectionURL.getParameter("nacos.group", DEFAULT_GROUP);
+        // Compatible with nacos grouping via group.
+        String group = connectionURL.getParameter(GROUP_KEY, DEFAULT_GROUP);
```

Comment:
Use `dubbo` as the default group will be better?

Reply:
If it is modified to dubbo, then the nacos registration center and configuration center side also need to be adjusted synchronously. This impact is relatively large.